test_migrations/test1_pg/2_create_index_function_if_not_exists_up.sql

```diff
@@ -0,0 +1,23 @@
+-- this function allows us to create indexes if they don't exist
```

Comment:
This was mostly because I couldn't tell how the test case made the previous code fail.